### PR TITLE
Clear the debounce

### PR DESCRIPTION
**Pull request by jridgewell**
Supersedes #1542.

### CODE REVIEW

File: underscore.js [hunk 1]
```diff
@@ -870,6 +870,13 @@
 
       return result;
     };
+
+    debounced.clear = function () {
```
Comment (akre54) [inline underscore.js]: Looks like this is failing lint.

**Comment by akre54** [comment]
:+1:

File: underscore.js (hunk 1 above)
Comment (jridgewell) [inline underscore.js]: Fixed.

**Comment by jdalton** [comment]
This would be good to add to `_.throttle` too.
Lodash gets this as `throttle` is implemented by way of `_.debounce`.

**Comment by jridgewell** [comment]
Already working on it.

**Pull request merged**